### PR TITLE
Dsg 2121/form builder bread crumb

### PR DESCRIPTION
**Pull request by g8shen**
# Basics

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Styling and layout changes
- [ ] Code style changes (formatting, local variables)
- [ ] Refactoring (no functional changes)
- [ ] Build or CI related changes
- [ ] Tests
- [ ] Other... Please describe:

## Screenshot of the affected changes
Confirmation:

https://github.com/Nuvalence/dsgov-web/assets/39806680/5b60da82-d972-4ed5-a411-586bdfa01c68

Confirmation:

https://github.com/Nuvalence/dsgov-web/assets/39806680/fcd70a6d-86cc-499e-ab6d-bfae1c3cc146

Back To Notes And Claims:

https://github.com/Nuvalence/dsgov-web/assets/39806680/6eef5a7f-5cd4-4699-9d3e-060462e6ddd8

Schemas:

https://github.com/Nuvalence/dsgov-web/assets/39806680/c473fcdb-38aa-44e8-a60a-2ff13e4c9e6f

Transaction Configurations:

https://github.com/Nuvalence/dsgov-web/assets/39806680/49f1d8a5-7782-4119-8542-cb5ace13a1bb

New Transaction:

https://github.com/Nuvalence/dsgov-web/assets/39806680/5cfb448e-1bbf-470d-9f9c-f1c77fcf9027

## Checklist

- [ ] I have updated the documentation accordingly
- [ ] I have confirmed all acceptance criteria defined by the story have been met
- [ ] I have confirmed that Sonar Qube findings for new code are reviewed and fixed, or will be whitelisted

### CODE REVIEW

File: libs/public/feature/dashboard/src/lib/components/transaction-routing/transaction-routing.component.ts [hunk 1]
```diff
@@ -2,8 +2,8 @@ import { CommonModule } from '@angular/common';
 import { ChangeDetectionStrategy, ChangeDetectorRef, Component, OnDestroy, OnInit } from '@angular/core';
 import { ActivatedRoute, NavigationEnd, NavigationSkipped, Router, RouterModule } from '@angular/router';
 import { FormRendererComponent, FormTransactionService } from '@dsg/shared/feature/form-nuv';
-import { NuverialBreadcrumbComponent, NuverialSnackBarService, NuverialSpinnerComponent } from '@dsg/shared/ui/nuverial';
-import { catchError, EMPTY, filter, switchMap, take, tap } from 'rxjs';
+import { INuverialBreadCrumb, NuverialBreadcrumbComponent, NuverialSnackBarService, NuverialSpinnerComponent } from '@dsg/shared/ui/nuverial';
+import { EMPTY, catchError, filter, switchMap, take, tap } from 'rxjs';
```
Comment (kevincheng0) [inline libs/public/feature/dashboard/src/lib/components/transaction-routing/transaction-routing.component.ts:6]: `EMPTY`, `catchError` don't seem to be used

File: libs/agency/feature/admin/src/lib/components/schemas-list/schema-form/schema-form.component.html [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <ng-container>
-  <nuverial-breadcrumb class="full-container" (click)="navigateToSchemas()" breadcrumbTitle="Back to Schemas"></nuverial-breadcrumb>
+  <nuverial-breadcrumb class="full-container-background" [breadCrumbs]="breadCrumbs"></nuverial-breadcrumb>
```
Comment (derekdiep42) [inline libs/agency/feature/admin/src/lib/components/schemas-list/schema-form/schema-form.component.html:2]: I'm not sure you should touch the full-container/full-container-background stuff. I think souradeep has a PR incoming for that

File: libs/public/feature/dashboard/src/lib/components/transaction-routing/transaction-routing.component.ts (hunk 1 above)
Comment (g8shen) [reply libs/public/feature/dashboard/src/lib/components/transaction-routing/transaction-routing.component.ts:6]: Screenshot 2023-10-23 at 9 39 31 AM
Thanks for the catch! I think they are used in the file I just didn't change them so it didn't come up in the PR. I think the change of order is just my prettier re-ordering them :)

File: libs/shared/ui/nuverial/src/lib/components/breadcrumb/breadcrumb.component.scss [hunk 3]
```diff
@@ -44,6 +56,18 @@
 }
 
 :host(.full-container) {
+  .nuverial-breadcrumb {
```
Comment (derekdiep42) [inline libs/shared/ui/nuverial/src/lib/components/breadcrumb/breadcrumb.component.scss:59]: I don't think we should change this, did you test what it looks like in the public portal?

I think we leave alone these namings/changes here for now.

File: libs/agency/feature/admin/src/lib/components/schemas-list/schema-form/schema-form.component.html (hunk 2 above)
Comment (derekdiep42) [reply libs/agency/feature/admin/src/lib/components/schemas-list/schema-form/schema-form.component.html:2]: Confirmed that the public portal still uses the fullscreen css and is working fine. Resolving.

File: libs/shared/feature/form-nuv/src/lib/components/forms/builder/form-builder.component.spec.ts [hunk 4]
```diff
@@ -108,14 +122,25 @@ describe('FormBuilderComponent', () => {
...
         components: FormioConfigurationTestMock,
         display: 'wizard',
       });
     });
+
+    it('should call getFormConfigurationByKey and set BreadCrumb', async () => {
+      const { component } = await getFixture({ componentProperties: { breadCrumbs: mockBreadCrumbs } });
+
+      const getFormConfigurationByKeySpy = jest
+        .spyOn(component['_workApiRoutesService'], 'getTransactionDefinitionByKey$')
```
[the remark this answers is not in the document]
Comment (g8shen) [reply libs/shared/feature/form-nuv/src/lib/components/forms/builder/form-builder.component.spec.ts:137]: The rest of the tests in the file are being implemented this way so I was just following the pattern

**Review by derekdiep42 — COMMENTED** [review]
Looks good to me, but we should get a quick review from @Ortega-Dan or @jgnuvalence 👀

**Review by jgnuvalence — APPROVED** [review]
LGTM

**Pull request merged**